src/pages/achievements/index.tsx

```
@@ -334,7 +334,7 @@ const Component: FC = () => {
         <div className="boost">
           <p>{t(constantKeys.BOOST_YOUR_EAENINGS)}</p>
           <div className="plans">
-            <a href="https://vultisig.com/download/vultisig" target="_blank">
+            <a href="https://vultisig.com/downloads" target="_blank">
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)